osu.Game/Overlays/ChatOverlay.cs

```diff
@@ -237,10 +237,7 @@ private void load(OsuConfigManager config, OsuColour colours, TextureStore textu
             Schedule(() =>
             {
                 // TODO: consider scheduling bindable callbacks to not perform when overlay is not present.
-                channelManager.JoinedChannels.CollectionChanged += joinedChannelsChanged;
-
-                foreach (Channel channel in channelManager.JoinedChannels)
-                    ChannelTabControl.AddChannel(channel);
+                channelManager.JoinedChannels.BindCollectionChanged(joinedChannelsChanged, true);
```

Comment:
Note: There's no special reason why this was this way. `BindCollectionChanged` was added after the event.